src/js/components/PackageUpgradeOverview.js

```diff
@@ -1,135 +0,0 @@
-import React from 'react';
```

Comment:
@jfurrow  - are we sure we want to kill this file? I know there is some upgrades related code that sneaked into master instead of the feature branch. I see that you wrote this and don't want to remove it if it's needed for upgrades